src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/PartyServiceTest.java

```diff
@@ -108,6 +108,39 @@ void givenACcdCallbackContestedCase_whenStartEventCalledAndAllPartiesAreDigitalA
         Assertions.assertEquals(2, partiesOnCase.getValue().size(), "selected parties");
     }
 
+    @Test
+    void givenACcdCallbackContestedCase_whenAllPartiesUnrepresentSelected_thengetUnrepresentedParties() {
+        FinremCallbackRequest finremCallbackRequest = buildCallbackRequest();
+        FinremCaseDetails caseDetails = finremCallbackRequest.getCaseDetails();
+        FinremCaseData data = caseDetails.getData();
+        data.setCcdCaseType(CONTESTED);
+        data.getContactDetailsWrapper().setApplicantFmName("Tony");
+        data.getContactDetailsWrapper().setApplicantLname("B");
+
+        data.getContactDetailsWrapper().setRespondentFmName("Tony");
+        data.getContactDetailsWrapper().setRespondentLname("C");
+
+        List<DynamicMultiSelectListElement> dynamicElementList = List.of(getDynamicElementList(CaseRole.APP_SOLICITOR.getCcdCode()),
+            getDynamicElementList(CaseRole.RESP_SOLICITOR.getCcdCode()));
+
+        DynamicMultiSelectList parties = DynamicMultiSelectList.builder()
+            .value(dynamicElementList)
+            .listItems(dynamicElementList)
+            .build();
+
+        data.setPartiesOnCase(parties);
```

Comment:
What's the purpose of setting this?

Reply:
No purpose - was copied from above test. Removed now